### PR TITLE
add magnum-capi-helm driver to base footer

### PR DESCRIPTION
**Pull request by scrungus**
This patch enables the addition of the new driver into the magnum container

### CODE REVIEW

**Comment by scrungus** [comment]
We don't really need to wait for AIOs to run for this change as magnum isn't currently included in that environment

**Review by JohnGarbutt — DISMISSED** [review]
It would be great to get this in, so we can build some fresh containers to test out.

**Comment by Alex-Welsh** [comment]
> It would be great to get this in, so we can build some fresh containers to test out.

You should be able to build containers from any branch, so could test before merging. We have it as a build option on the [workflow](https://github.com/stackhpc/stackhpc-kayobe-config/actions/workflows/stackhpc-container-image-build.yml) .

**Comment by scrungus** [comment]
https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6378828585

**Comment by scrungus** [comment]
@Alex-Welsh can you publish to prod a container from any branch?

**Comment by Alex-Welsh** [comment]
> @Alex-Welsh can you publish to prod a container from any branch?

You should be able to manually run the promotion workflow with any tags you want

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -33,7 +33,9 @@ kayobe_image_tags:
   kibana:
     centos: yoga-20230907T152300
   magnum:
-    rocky: yoga-20230825T142202
+    rocky: yoga-20231002T101716
```
Comment (mnasiadka) [inline etc/kayobe/kolla/globals.yml]: Since we use one tag - can you put it in L12?

**Comment by mnasiadka** [comment]
A reno would also be nice

File: releasenotes/notes/magnum-capi-driver-2dd8cf6e68a1919c.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds the new capi driver into the magnum container
```
Comment (JohnGarbutt) [inline releasenotes/notes/magnum-capi-driver-2dd8cf6e68a1919c.yaml]: Nit: I feel this should really link to your docs on how to install the magnum capi driver? Did we get that merged anywhere yet?

**Review by JohnGarbutt — DISMISSED** [review]
We can tweak the release notes later, lets try get this in so we can merged into Scientific OpenStack ASAP.

**Comment by scrungus** [comment]
tested in aio

**Review by JohnGarbutt — APPROVED** [review]
OK, thank you.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**